plugin/xcodebuild.vim

```diff
@@ -98,7 +98,7 @@ function! s:project_file()
 endfunction
 
 function! s:scheme()
-  return '-scheme '. s:scheme_name()
+  return '-scheme \"'. s:scheme_name() . '\"'
```

Comment:
I think this could be `shellescape(s:scheme_name())` instead of manually adding quotes.

```text
shellescape({string} [, {special}])			*shellescape()*
		Escape {string} for use as a shell command argument.
		On MS-Windows and MS-DOS, when 'shellslash' is not set, it
		will enclose {string} in double quotes and double all double
		quotes within {string}.
		For other systems, it will enclose {string} in single quotes
		and replace all "'" with "'\''".
```